src/util/Fetcher.ts

```diff
@@ -32,6 +32,9 @@ export interface IFetcher {
 export default class Fetcher implements IFetcher {
   async fetch(url: RequestInfo | URL, init?: RequestInit): Promise<Response> {
     const fetchUrl = url instanceof URL ? url.toString() : url;
+    if (typeof window !== "undefined" && typeof window.fetch !== "undefined") {
+      return window.fetch(fetchUrl, init);
+    }
```

Comment:
I would rather make this decision at the point where `_fetch` is defined. So redefine an internal `fetch` at the top level, and do this check only once. Also, `typeof window.fetch === "function"`

Reply:
I merged this PR a bit prematurely, and forgot to add this. https://github.com/inrupt/solid-auth-fetcher/pull/183 addresses this comment.